dist/form-schema.js

```diff
@@ -152,6 +152,7 @@ function (_FormFieldSchema) {
       this.addSystemColumn('Created Duration', 'createdDuration', '_created_duration', 'integer');
       this.addSystemColumn('Updated Duration', 'updatedDuration', '_updated_duration', 'integer');
       this.addSystemColumn('Edited Duration', 'editedDuration', '_edited_duration', 'integer');
+      this.addSystemColumn('Record Key', 'recordKey', '_record_key', 'string');
```

Comment:
did record sequence need to be added here as well?

Reply:
No, I already have access to those attributes without adding those columns, in fact, I am going to remove those additions, thank you